File: .sdk/tm/rb/utility/make_context.rb
# IpAddressLookupTwo SDK utility: make_context
require_relative '../core/context'
module IpAddressLookupTwoUtilities
  MakeContext = ->(ctxmap, basectx) {
    IpAddressLookupTwoContext.new(ctxmap, basectx)
  }
end

File: py/ipaddresslookuptwo_sdk/utility/make_context.py
# IpAddressLookupTwo SDK utility: make_context

from ipaddresslookuptwo_sdk.core.context import IpAddressLookupTwoContext


def make_context_util(ctxmap, basectx):
    return IpAddressLookupTwoContext(ctxmap, basectx)
